tic-tac-toe/src/utils.tsx

```diff
@@ -0,0 +1,45 @@
+import colors from "./colors.json";
+
+export type Symbol = "X" | "O";
+export type Slot = Symbol | null;
+export type Score = {X: number, O: number};
+export type Winner = Slot | 'DRAW';
+
+export function getSVG(symbol: Symbol): any {
```

Comment:
Interesting way to approach this lol. I usually use an `Icon` component but this works too I guess